.github/workflows/package-test.yml

```diff
@@ -45,9 +45,14 @@ jobs:
     - run: yarn build-report
     - run: sudo apt-get install xvfb
 
-    # Disabled to fix CI while we investigate what the bug is.
-    #   perf-diagnostics-third-party, oopif-requests, oopif-scripts
-    - run: xvfb-run --auto-servernum bash $GITHUB_WORKSPACE/lighthouse-core/scripts/release/package-test.sh --fraggle-rock --invert-match perf-diagnostics-third-party oopif-requests oopif-scripts
+    - name: Define ToT chrome path
+      run: echo "CHROME_PATH=/home/runner/chrome-linux-tot/chrome" >> $GITHUB_ENV
+
+    - name: Install Chrome ToT
```

Comment:
should this (and the other cases) include a TODO to eventually move back to stable?

Also, what's left testing stable?

Reply:
- Bundled Lighthouse (Legacy navigation)
- Package test (Legacy navigation)
- Chrome stable smokes from the standard matrix

Reply:
> should this (and the other cases) include a TODO to eventually move back to stable?

TBH I'm fine shifting everything (except our explicitly Chrome stable smokes) to use ToT